src/table.tsx

```diff
@@ -75,83 +127,207 @@ export const Table = React.forwardRef(
-                                {row.map((cell, idx) => (
-                                    <td key={idx} className={styles.verticalAlign[rowVerticalAlign]}>
-                                        {/**
-                                         * In collapsedRowsMode, we render the row heading text before every cell content, except for the first cell
-                                         * of every row, which is rendered with a medium weight font, as it's the row title.
-                                         * */}
-                                        {idx !== 0 && collapsedRowsMode && (
-                                            // this is aria-hidden because screen readers already read the column heading from the th
-                                            <div className={styles.mobileCellHeading} aria-hidden>
-                                                <Text1 medium color={vars.colors.textSecondary}>
-                                                    {heading[idx]}
-                                                </Text1>
-                                            </div>
-                                        )}
-
-                                        <Text2
-                                            as="div"
-                                            weight={idx === 0 && collapsedRowsMode ? 'medium' : 'regular'}
-                                            wordBreak={false}
+                    {content.length > 0 ? (
+                        content.map((row, rowIdx) => {
+                            const rowCells = !Array.isArray(row) ? row.cells : row;
+                            const rowActionsList = !Array.isArray(row) ? row.actions ?? [] : [];
+
+                            const actions = (
+                                <Inline space={16}>
+                                    {rowActionsList.map((action, index) => {
+                                        if ('Icon' in action || 'checkedProps' in action) {
+                                            // action is a CellAction object
+                                            return <CellActionIconButton key={index} {...action} />;
+                                        }
+                                        // action is a React.ReactElement
+                                        return action;
+                                    })}
+                                </Inline>
+                            );
+
+                            // buttons + inline space
+                            const actionsElementWidth = rowActionsList.length
+                                ? `calc(${iconContainerSize.small} * ${rowActionsList.length} + 16px * ${
+                                      rowActionsList.length - 1
+                                  })`
+                                : '0px';
+
+                            return (
+                                <tr key={rowIdx} style={{position: 'relative'}}>
```

Comment:
add a comment explaining this `position: relative`

Reply:
done